### DISCUSSION TITLE
Show sector times of fastest lap under current sector times

### DISCUSSION THREAD

**Discussion by Justmie** [f1multiviewer/issue-tracker#408] [Feature requests] [answered]
As the title shows, would it be possible to make an extra row in the livetiming under the current sector times, that shows the sector times of the fastest lap of that driver.

Something like this with Alonso's timing:
![Untitled](https://github.com/f1multiviewer/issue-tracker/assets/73039009/c307c907-f2eb-4eb1-b81f-ecb60614d706)

**Comment by f1multiviewer** [comment, marked as answer]
You already can enable this in settings
